app/views/ExploreDeep/ActionCell/ProjectJoinModal/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useCallback } from 'react';
+import {
+    _cs,
+    isDefined,
+} from '@togglecorp/fujs';
+import {
+    Modal,
+    Button,
+    TextArea,
+} from '@the-deep/deep-ui';
+import {
+    ObjectSchema,
+    useForm,
+    requiredStringCondition,
+    getErrorObject,
+    lengthGreaterThanCondition,
+} from '@togglecorp/toggle-form';
+
+import styles from './styles.css';
+
+interface ProjectJoinFields {
+    reason: string;
+}
+
+type FormType = Partial<ProjectJoinFields>;
+type FormSchema = ObjectSchema<FormType>;
+type FormSchemaFields = ReturnType<FormSchema['fields']>;
+
+const schema: FormSchema = {
+    fields: (): FormSchemaFields => ({
+        reason: [
+            requiredStringCondition,
+            lengthGreaterThanCondition(100),
```

Comment:
do we need this condition ?

Reply:
@AdityaKhatri  did this. Can you please answer this question?

Reply:
This is required